chapters/210_digital-identity-personal-data-ownership.md

```diff
@@ -3,51 +3,50 @@
 [@whitepaper_2012_the-value-of-our-digital-identity_definition]. *Personal information*, in this 
 case, refers to inherent (e.g. date of birth) as well as imposed (e.g. credit card number) 
-characteristics. The individual to whom those data relates to, is therewith the owner of that  
+characteristics. The individual to whom those data relates to, is the owner of that  
 *Digital Identity*.
 From a technical standpoint, a *Digital Identity* is essentially a collection of characteristics, 
 attributes and time series data (e.g. interaction logs or bank transfer history). Based on a subset 
 of those attribute values, a unique fingerprint can be easily generated. Depending on the data point 
-and complexity of its value, either a unique identifier on its own (e.g. social security number) - 
-depending on the context - or only a few are also enough to generate such fingerprint. Hence it 
+and complexity of its value, such a fingerprint could require either a single, unique identifier on its own (e.g. social security number) or only a few. Hence, it 
 doesn't take a 
 complete set of attributes including all its values, but rather just a fraction of a *Digital 
 Identity* in order to determine its rightful owner and physical counterpart. The *Digital Identity* 
 can be viewed as an avatar in digital environments or even as the digital part of a persons's 
-identity. That is, a *Digital Identity* of a living individual can't just be reduced to some 
-bits and bytes, instead it should be valued as an appropriate, and maybe even legal, representation 
+identity. That is, a *Digital Identity* of a living individual cannot simply be reduced to 
+bits and bytes. Instead, it should be valued as an appropriate, and perhaps legal, representation 
 in certain contexts and for a variety of purposes.
 In some of those situations it might be required (e.g. administrative purposes) to ensure a certain 
 level of authenticity for a *Digital Identity* or for particular attributes of it. This means, to 
 provide reliable confirmation that the attribute values are really the ones that belong to exactly 
-that individual they pretend to belong. An independent third party, who is trusted by all entities 
-participating in such a construct, could somehow verify (or vouch) the subject in question.
-On the other hand this concept opens up at least on class of attack scenarios. The risk of identity 
-theft for example increases dramatically, when assigning such value to a *Digital Identity*, 
-because the attacker is now longer required to be physically present in order to impersonate that
-identity or "steal" certain unique identifiers from person. Instead it's sufficient to gain access 
-to the areas where those sensitive data is stored.
-It is noted, that different technical solutions to these issues do exist and will be discussed 
+they individual they supposedly belong to. An independent third party, who is trusted by all entities 
```

Comment:
@jag82 Did you want to write "the" instead of the first "they". currently it says "the attribute values are really the ones that belong to exactly __they__ individual they supposedly belong to". or maybe "their"?